modules/gapi/src/api/gbackend.cpp

```diff
@@ -411,6 +411,12 @@ void createMat(const cv::GMatDesc &desc, cv::Mat& mat)
     {
         GAPI_Assert(!desc.planar);
         mat.create(desc.dims, desc.depth);
+#if !defined(GAPI_STANDALONE)
```

Comment:
Similar to https://github.com/opencv/opencv/blob/4.x/modules/gapi/src/backends/common/gbackend.hpp#L26